recipes/mono/mono-osx.recipe

```diff
@@ -0,0 +1,149 @@
+        if not os.path.exists(mono_prefix_dir):
+            m.error("ERROR: Mono not found!")
+            raise FatalError("Mono not found!")
+        for f in self._mono + self._monoscripts:
+            src = os.path.join(mono_prefix_dir, f)
+            dest = os.path.join(self.config.prefix, f)
+            if os.path.isdir(src):
+                shell.copy_dir (src, dest)
+            elif os.path.isfile(src):
+                if not os.path.exists(os.path.dirname(dest)):
+                    os.makedirs(os.path.dirname(dest))
+                shutil.copy(src, dest)
+            else:
+                print "WARNING: %s not found" % f
+
+        #replace mono scripts
+        for f in self._monoscripts:
+            script = os.path.join(self.config.prefix, f)
+            self.inplace_change(script, mono_prefix_dir, self.config.prefix)
+
+        monosgen_link = os.path.join(self.config.prefix, 'lib', 'libmonosgen-2.0.dylib')
+        mono_link = os.path.join(self.config.prefix, 'lib', 'libmono-2.0.dylib')
+
+        shutil.move(os.path.join(self.config.prefix, 'bin', 'mono-sgen64'),
+                    os.path.join(self.config.prefix, 'bin', 'mono-sgen'))
+        shutil.copy(os.path.join(self.config.prefix, 'bin', 'mono-sgen'),
+                    os.path.join(self.config.prefix, 'bin', 'mono'))
+        if(os.path.exists(mono_link)):
+            os.remove(mono_link)
+        if(os.path.exists(monosgen_link)):
+            os.remove(monosgen_link)
+        os.symlink(os.path.join(self.config.prefix, 'lib', 'libmonosgen-2.0.1.dylib'),
+                   monosgen_link)
+        os.symlink(monosgen_link,
+                   mono_link)
+
+    def inplace_change(self, filename, old_string, new_string):
+        # Safely read the input filename using 'with'
+        with open(filename) as f:
+            s = f.read()
+            if old_string not in s:
+                print '"{old_string}" not found in {filename}.'.format(**locals())
+                return
+
+        # Safely write the changed content, if found in the file
+        with open(filename, 'w') as f:
+            print 'Changing "{old_string}" to "{new_string}" in {filename}'.format(**locals())
+            s = s.replace(old_string, new_string)
+            f.write(s)
+    
+    def post_install(self):
+        config = os.path.join(self.config.prefix, 'etc', 'mono', 'config')
+        prefix = self.config.libdir
+        if prefix[-1] != '/':
+            prefix += '/'
+        shell.replace(config, {prefix: '',
+            'libintl.dylib': 'libintl.8.dylib',
+            'libintl.so': 'libintl.8.so',
+            'libgdiplus.dylib': 'libgdiplus.0.dylib',
+            'libgdiplus.so': 'libgdiplus.so.0',
+            })
+        shell.call ('rm -rf lib/mono/2.0/nunit*', self.config.prefix)
+        shell.call ('rm -rf lib/mono/4.5/nunit*', self.config.prefix)
+        shell.call ('rm -rf lib/mono/gac/*/2.4.8.0__96d09a1eb7f44a77',
```

Comment:
These are copied from the old mono recipe, right? Shouldn't they be updated to 5.0?